scene/gui/text_edit.cpp

```diff
@@ -416,9 +416,10 @@ void TextEdit::_click_selection_held() {
 
 void TextEdit::_update_selection_mode_pointer() {
 	Point2 mp = Input::get_singleton()->get_mouse_position() - get_global_position();
+	Vector2 vs = get_viewport_transform().get_scale() *  get_global_transform().get_scale();
```

Comment:
Your PR has a minor style error here, as Travis noted. Remove the second space after the `*`.

Reply:
Done